### PR TITLE
#1189 #1199 fix typo and add spectral volumes rst

### LINKED ISSUE #1199: typo in thermal models notebook
The boundary conditions at the tabs in the pouch cell models are written incorrectly in the notebook

### PR DESCRIPTION
**Pull request by rtimms**
# Description

Fix some typos in the thermal models notebook and add rst file for spectral volume docs

Fixes #1199 and #1189 

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/master/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [ ] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: examples/notebooks/models/thermal-models.ipynb [hunk 1]
```diff
@@ -196,16 +198,16 @@
...
-    "$$ -\\lambda_{eff} \\nabla_\\perp T \\cdot \\boldsymbol{n} = h_{cp}(T-T_\\infty),$$  \n",
+    "$$ -\\lambda_{eff} \\nabla_\\perp T \\cdot \\boldsymbol{n} = \\frac{(L_{cn}+L_n+L_s+L_p)h_{edge}+L_{cp}h_{cp}}{L_{cn}+L_n+L_s+L_p+L_{cp}}(T-T_\\infty),$$  \n",
     "at the positive tab, and\n",
     "$$ -\\lambda_{eff} \\nabla_\\perp T \\cdot \\boldsymbol{n} = h_{edge}(T-T_\\infty),$$  \n",
     "elsewhere.\n",
     "\n",
     "Here the heat source term is averaged in the x direction so that $\\bar{Q}=\\bar{Q}(y,z)$. The parameter $\\lambda_{eff}$ is the effective thermal conductivity, computed as \n",
     "\n",
-    "$$ \\rho_{eff} = \\frac{\\sum_k \\rho_k c_{p,k} L_k}{\\sum_k L_k}.$$\n",
+    "$$ \\lambda_{eff} = \\frac{\\sum_k \\lambda_k L_k}{\\sum_k L_k}.$$\n",
```
Comment (tinosulzer) [inline examples/notebooks/models/thermal-models.ipynb:210]: should we change the name of the parameter in pybamm accordingly?

**Review by tinosulzer — APPROVED** [review]
thanks @rtimms

File: examples/notebooks/models/thermal-models.ipynb (hunk 1 above)
Comment (rtimms) [reply examples/notebooks/models/thermal-models.ipynb:210]: the name is correct in pybamm - I had just written completely the wrong equation in the notebook!

Comment (rtimms) [reply examples/notebooks/models/thermal-models.ipynb:210]: looks like I copied it from above to save me writing out the sum again and forgot to change rho to lambda

**Pull request merged**